Compute/src/V1/Gapic/AcceleratorTypesGapicClient.php

```diff
@@ -41,7 +43,8 @@
 use Google\Cloud\Compute\V1\ListAcceleratorTypesRequest;
 
 /**
- * Service Description: Services.
+ * Service Description:
+ * Services
```

Comment:
🤔  

Why this change? Is it intentional?

Reply:
This newline is part of the proto's comment, which reads as below:
```
//
// Services
// 
// The AcceleratorTypes API.
```